packages/router/src/router.js

```diff
@@ -55,13 +55,13 @@ const PrivatePageLoader = ({
   ) {
     return children
   } else {
-    return (
-      <Redirect
-        to={`${unauthenticatedRoute()}?redirectTo=${
+    const redirectUrl = window
```

Comment:
Would undefined be OK instead of null?

Then you could use `window?.location?.pathname|search`

Reply:
Sure